flask_s3.py

```diff
@@ -39,6 +41,48 @@
 
 __version__ = (0, 2, 10)
 
+FLASKS3_PREFIX_DEFAULT = ''
+FLASKS3_VERSION_FUNCTION_DEFAULT = None
+
+
+def _get_versioning_prefix(app):
```

Comment:
Thank you for all the feedback. 
Yes, you are right. For my use case you can set a default prefix and before 
calling `flasks3_instance.init_app(app)` just add a version string to that prefix.

I'll squash these commits once you think this is ready to be merged.